tests/Agent/IntegrationTests/Applications/WebApiAsyncApplication/Controllers/AsyncFireAndForgetController.cs

```diff
@@ -63,14 +61,12 @@ public async Task<string> Async_FireAndForget()
             //WITHOUT
             //  AF-* Not sure, probably an error
 
-            return "Worked";
+            return Task.FromResult("Worked");
         }
 
         [HttpGet]
         [Route("AsyncFireAndForget/Async_Sync")]
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-        public async Task<string> Async_Sync()
-#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+        public Task<string> Async_Sync()
```

Comment:
Are we sure that this is still testing the scenario it was designed to?  The name of the method being "Async_Sync()" leads me to think that this needs to be an async method to meet the goals of the test.

Reply:
That's a good question. Our instrumentation looks for the property `IsAsync` to be set to true, and adjusts its behavior. The `IsAsync` property is only true when we're in an `async` method I believe. I don't think that it's enough for a method to just return a `Task` to trigger that behavior.